src/storybook-components/EventSearchPage.vue

```diff
@@ -0,0 +1,130 @@
+<template>
+  <rpl-base-layout class="event-search-page-demo">
+
+    <template slot="header">
+      <rpl-site-header
+        :logo="mock.header.logo"
+        :links="mock.header.links"
+        :breakpoint="mock.header.breakpoint"
+        :searchTerms="mock.header.searchTerms"
+        showSearch
+        sticky
+        @open="menuOpenFunc"
+        @search="searchFunc"
+      />
+    </template>
+
+    <rpl-page-layout
+      :sidebar="sidebar"
+      class="main rpl-container"
+    >
+      <template slot="aboveContent">
+        <rpl-search-form
+          :title="mock.eventSearchForm.title"
+          :searchPlaceholder="mock.eventSearchForm.searchPlaceholder"
+          :prefillSearchTerm="mock.eventSearchForm.prefillSearchTerm"
+          :filterForm="mock.eventSearchForm.filterForm"
+          :theme="mock.eventSearchForm.theme"
+          @search="getSearchResults"
+          class="rpl-site-constrain--on-all"
+        />
+        <rpl-divider />
+      </template>
+
+      <rpl-container class="demo-main">
```

Comment:
Hey @Ahmedjabar You don't need `rpl-container` here as we have it been used in `rpl-page-layout` already. It will bring some issue when we use it nested at the moment. You can use `div` instead if you need a container here.